src/Components/Header/Header.js

```diff
@@ -89,9 +86,7 @@ const Header = ({ handleTextfieldChange }) => {
               open={isLanguageSelectOpen}
               onOpen={handleOpenLanguage}
               onClose={handleCloseLanguage}
-              IconComponent={LanguageIcon}
-              renderValue={() => setRenderValue()}
-              sx={{ '& .MuiSvgIcon-root': { right: '1.5rem', color: '#FFFFFF' } }}
+              sx={{ '& .MuiSvgIcon-root': { color: '#FFFFFF' } }}
```

Comment:
Out of curiosity, do we need this line?

Reply:
Yes, this line makes the Select chevron icon white.